### PR TITLE
Disable documentTitle in NavigationRoot component

### LINKED ISSUE #2636: Favicon - Flashing NEW! Expensify.cash and icon without white background can be seen in tab
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Expected Result:
I expected no flashing with NEW! Expensify.cash and switching with Expensify.cash

## Actual Result:
Flashing with NEW! Expensify.cash and switching with Expensify.cash

## Action Performed:
1. Launch the app as Account B and log in
2. As account B, send some messages to account A
3. Go to https://staging.expensify.cash as Account A in Web/Chrome
3. Observe the favicons and NEW! Expensify.cash

## Workaround:
No need, visual issue. 

## Platform:
Where is this issue occurring?

 Web ✔️ 
 iOS
 Android
 Desktop App
 Mobile Web

**Version Number:** 1.0.34-0

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/44479856/116629484-5a5c2280-a91f-11eb-94a3-d47c397dc0dd.mp4

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by deetergp**
[Mu](https://github.com/Expensify/Expensify.cash/issues/2636#issuecomment-843636746)

### Details
The listener that sets the page title when there are new chat items, that gets attached in `AuthScreens` was getting overridden by the `NavigationContainer` in NavigationRoot.js, which apparently mounts a few more times after the listener updates the title. This changes it so that the `NavigationContainer` only sets the title to `Expnesify.cash` when the `document.title` is not already set.

cc @marcaaron 

### Fixed Issues

Fixes https://github.com/Expensify/Expensify.cash/issues/2636

### Tests/QA
1. Launch the app as Account B and log in
2. As account B, send some messages to account A
3. Go to localhost:8080 as Account A in Web/Chrome
4. Observe the favicons and NEW! Expensify.cash
5. Verify that once `(NEW!) Expensify.cash` is set, it stays set.

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

#### Web
https://user-images.githubusercontent.com/766197/118739217-bbc83080-b7fd-11eb-9fdf-fe1e3c9565eb.mp4

### CODE REVIEW

File: src/libs/Navigation/NavigationRoot.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ class NavigationRoot extends Component {
                 ref={navigationRef}
                 linking={linkingConfig}
                 documentTitle={{
-                    formatter: () => 'Expensify.cash',
+                    formatter: () => `${document.title ?? 'Expensify.cash'}`,
```
Comment (marcaaron) [inline src/libs/Navigation/NavigationRoot.js:43]: Could we just nix the whole `documentTitle` prop?

Comment (deetergp) [reply src/libs/Navigation/NavigationRoot.js:43]: Then it changes the title to `Report` 😂

Comment (marcaaron) [reply src/libs/Navigation/NavigationRoot.js:43]: How about just disabling it?
https://reactnavigation.org/docs/navigation-container/#documenttitle

```jsx
documentTitle={{
    enabled: false,
}}
```

Comment (deetergp) [reply src/libs/Navigation/NavigationRoot.js:43]: That appears to work just fine. I'll change it.

**Comment by deetergp** [comment]
Updated.

**Review by marcaaron — APPROVED** [review]

**Pull request merged**